package.json

```diff
@@ -55,6 +55,8 @@
     "ethereumjs-util": "^7.1.5",
     "ethers": "5.6.9",
     "etherscan-api": "^10.0.5",
+    "lodash": "^4.17.20",
+    "moment": "^2.29.4",
```

Comment:
Moment is deprecated and no longer recommended for new projects. 
https://momentjs.com/docs/#/-project-status/